api/logs/views.py

```diff
@@ -97,6 +117,7 @@ class LogNodeList(JSONAPIBaseView, generics.ListAPIView, ODMFilterMixin):
 
     def get_queryset(self):
         log = self.get_log()
+        self.check_log_permission(log)
```

Comment:
We don't use the return value here, and `check_log_permission` above returns `True` or `False`. Does it also raise in the event that the user doesn't have permission?